src/com/owncloud/android/files/services/FileDownloader.java

```diff
@@ -80,6 +86,10 @@
     public static final String EXTRA_LINKED_TO_PATH = "LINKED_TO";
     public static final String ACCOUNT_NAME = "ACCOUNT_NAME";
 
+    public static final String KEY_FILE_REMOTE_PATH = "FILE_REMOTE_PATH";
+    public static final String KEY_ACCOUNT_NAME = "ACCOUNT_NAME";
```

Comment:
Sure, I will change the name to EXTRA_ACCOUNT_NAME but about declaring it in RetryDownloadJobService, there are other final variables such as EXTRA_REMOTE_PATH which is declared in FileDownloader but it's used where download broadcast is received.